tests/unittests/OperatorTest.cpp

```diff
@@ -1346,6 +1346,23 @@ TEST_P(InterpAndCPU, QuantizedArithmeticRescaled) {
   }
 }
 
+TEST_P(Operator, QuantizedTranspose) {
+  auto *A = mod_.createVariable(ElemKind::FloatTy, {2, 3}, "A",
+                                VisibilityKind::Public);
+  auto *B = mod_.createVariable(ElemKind::FloatTy, {3, 2}, "B",
+                                VisibilityKind::Public);
+  A->getPayload().getHandle() = {1, 1.2f, 0.5f, 1.3f, 2.7f, 5.8f};
+  B->getPayload().getHandle() = {1, 1.3f, 1.2f, 2.7f, 0.5f, 5.8f};
```

Comment:
Should we use `Tensor::transpose` here to get the payload of B instead of transposing by hand?

Reply:
+1. Maybe even make the transpose operation as a node on fp32 (A var). This way you'll test transposes on both types for openCL and other backends.

Reply:
use Tensor::transpose now. I also added a transpose operation on fp32 var A. It tests both types now.